pallets/gear/src/benchmarking/mod.rs

```diff
@@ -40,7 +40,9 @@ mod sandbox;
 use ::alloc::vec;
 
 mod syscalls;
+mod tests;
```

Comment:
And for other places where required
```suggestion
#[cfg(test)]
mod tests;
```

Reply:
discussed